src/components/moderation/moderation-common.ts

```diff
@@ -55,6 +55,15 @@ import { set_interval } from "../../utils/node.js";
 export const duration_regex = /(?:perm\b|(\d+)\s*([mhdwMys]))/;
 
 export const moderation_on_team_member_message: string = "Can't apply this moderation on team members";
+export const joke_responses: Array<string> = new Array<string>(
+    "You won't get off that easy! ;)",
+    "Try again next time lmao",
+    "Didn't work. Maybe a skill issue?",
+);
+
+function get_random_array_element(arr: any[]) {
```

Comment:
```suggestion
function get_random_array_element<T>(arr: T[]) {
```

Reply:
I'm getting issues with this change. The following error is what I receive that I do not receive with the old approach.

```
TS2345: Argument of type string | undefined is not assignable to parameter of type string
Type undefined is not assignable to type string
```

Reply:
Currently the function has an `any` return type, once you make the function more well-typed and TS sees it returns `T | undefined` then you'll run into problems later with `await this.reply_with_error(command, get_random_array_element(joke_responses));`, which is a good thing since TS just caught a potential bug. The easiest fix is just wrap `get_random_array_element(joke_responses)` in an `unwrap()`.

Reply:
Alternatively, maybe just assert the array isn't empty in the body of the function

Reply:
Decided to just go with the unwrap() approach as its the simplest for me to bring together.

Reply:
Awesome thanks!